Sources/FluentKit/Properties/Boolean.swift

```diff
@@ -41,9 +41,9 @@ extension Fields {
 ///         func revert(on database: Database) async throws -> Void { try await database.schema(MyModel.schema).delete() }
 ///     }
 ///
-/// - Note: See also ``OptionalBooleanProperty`` and ``BooleanPropertyFormat``.
+/// > Note: See also ``OptionalBooleanProperty`` and ``BooleanPropertyFormat``.
 @propertyWrapper
-public final class BooleanProperty<Model, Format>
+public final class BooleanProperty<Model, Format>: @unchecked Sendable
```

Comment:
Huh. Surprisingly enough, the `@unchecked Sendable` on `FieldProperty` seems to be sufficient to suppress the warning about the property wrapper's backing being mutable - so yeah, since `BooleanProperty` (and `OptionalBooleanProperty`) have no other mutable values and gets `Sendable` from `Property`, they don't need the separate declaration.

Reply:
`TimestampProperty`, `EnumProperty`, and `OptionalEnumProperty` also have this win.